modules/aws_k8s_base/tf_module/ingress_nginx.tf

```diff
@@ -60,6 +60,7 @@ resource "helm_release" "ingress-nginx" {
           "linkerd.io/inject" : "enabled"
           "config.linkerd.io/skip-inbound-ports" : "80,443" // NOTE: should be removed when this is fixed: https://github.com/linkerd/linkerd2/issues/4219
           "cluster-autoscaler.kubernetes.io/safe-to-evict" : "true"
+          "viz.linkerd.io/tap-enabled" : "true"
```

Comment:
super minor, lets move this up one line to keep the linkerd annotations in the same bloc